src/keyring/KeyringController.test.ts

```diff
@@ -508,6 +699,11 @@ describe('KeyringController', () => {
     expect(listenerUnlock.called).toBe(true);
   });
 
+  it('should return current seedphrase', async () => {
+    const seedPhrase = await keyringController.verifySeedPhrase();
+    expect(seedPhrase).toBeDefined();
```

Comment:
Is there any way for us to stub whatever is responsible for generating the seed phrase so we can be more exact with this assertion?

Reply:
same as above https://github.com/MetaMask/controllers/pull/693#discussion_r857401548